src/OrleansRuntime/ConsistentRing/VirtualBucketsRingProvider.cs

```diff
@@ -5,6 +5,8 @@
 
 namespace Orleans.Runtime.ConsistentRing
 {
+    using Orleans.Runtime.Configuration;
```

Comment:
The same note as in `Catalog`. `Should this be before or after namespace Orleans.Runtime? I'm note sure if we should maintain consitency of keeping namespace directives on the other scope.